api/users/serializers.py

```diff
@@ -32,36 +61,27 @@ class UserSerializer(JSONAPISerializer):
-                                                           allow_blank=True, help_text='Twitter Handle'), required=False, source='social.twitter')))
-    linkedin = DevOnly(HideIfDisabled(AllowMissing(ser.CharField(required=False, source='social.linkedIn',
-                                                            allow_blank=True, help_text='LinkedIn Account'), required=False, source='social.linkedIn')))
-    impactstory = DevOnly(HideIfDisabled(AllowMissing(ser.CharField(required=False, source='social.impactStory',
-                                                               allow_blank=True, help_text='ImpactStory Account'), required=False, source='social.impactStory')))
-    orcid = DevOnly(HideIfDisabled(AllowMissing(ser.CharField(required=False, source='social.orcid',
-                                                         allow_blank=True, help_text='ORCID'), required=False, source='social.orcid')))
-    researcherid = DevOnly(HideIfDisabled(AllowMissing(ser.CharField(required=False, source='social.researcherId',
-                                                      allow_blank=True, help_text='ResearcherId Account'), required=False, source='social.researcherId')))
-    researchgate = DevOnly(HideIfDisabled(AllowMissing(ser.CharField(required=False, source='social.researchGate',
-                                                      allow_blank=True, help_text='ResearchGate Account'), required=False, source='social.researchGate')))
-    academia_institution = DevOnly(HideIfDisabled(AllowMissing(ser.CharField(required=False, source='social.academiaInstitution',
-                                                      allow_blank=True, help_text='AcademiaInstitution Field'), required=False, source='social.academiaInstitution')))
-    academia_profile_id = DevOnly(HideIfDisabled(AllowMissing(ser.CharField(required=False, source='social.academiaProfileID',
-                                                      allow_blank=True, help_text='AcademiaProfileID Field'), required=False, source='social.academiaProfileID')))
-    baiduscholar = DevOnly(HideIfDisabled(AllowMissing(ser.CharField(required=False, source='social.baiduScholar',
-                                                           allow_blank=True, help_text='Baidu Scholar Account'), required=False, source='social.baiduScholar')))
-    ssrn = DevOnly(HideIfDisabled(AllowMissing(ser.CharField(required=False, source='social.ssrn',
-                                                           allow_blank=True, help_text='SSRN Account'), required=False, source='social.ssrn')))
     timezone = HideIfDisabled(ser.CharField(required=False, help_text="User's timezone, e.g. 'Etc/UTC"))
     locale = HideIfDisabled(ser.CharField(required=False, help_text="User's locale, e.g.  'en_US'"))
 
+    social = HideIfDisabled(ListLinksField(
+        {
+            'personal_website': 'personal_website_url',
```

Comment:
Okay, what if we do this:

```json
{
    "data": {
            "attributes": {
                "social": {
                    "github": ["brianjgeiger","centerforopenscience"],
                    "twitter": ["thefoodgeek","awesomeography"],
                    "personal_website":["https://thefoodgeek.com/","https://awesomeography.com/"],
                    [...]
                }
            }
        }
}
```
This way we don't have to do a lot of validation on things. If we want, we could go crazy and give a usage string, or we could just do it in the docs. If we did it inline, it would be something like, `'github': {'items':['brianjgeiger','centerforopenscience'], 'usage': 'https://github.com/{}/},`. This way it can just get updated in one place if, say, linkedin changes their URLs for profiles or something.